.travis.yml

```
@@ -2,14 +2,12 @@ language: python
 
 env:
     matrix:
-        # These ancient versions are linked to an old libgfortran, but that version
-        # isn't pinned in the package metadata
-        - PYTHON_VERSION=3.5
-          PACKAGES="cython=0.28 numpy=1.10.0 matplotlib=1.5.1 nose proj4=4.9.1 scipy=0.16.0 libgfortran=1 owslib=0.19.1"
-          PYTHONHASHSEED=0  # So pytest-xdist works.
+        - NAME="Minimum dependencies."
+          PYTHON_VERSION=3.7
+          PACKAGES="cython=0.28.5 numpy=1.16 matplotlib=2.2.3 proj4=5.2.0 scipy=1.2.0 owslib=0.17"
```

Might was well sort these at the same time.
```suggestion
          PACKAGES="cython=0.28.5 matplotlib=2.2.3 numpy=1.16 owslib=0.17 proj4=5.2.0 scipy=1.2.0"
```